### PR TITLE
Abhsag/schedulebutton

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/gui/mainGui.py [hunk 1]
```diff
@@ -92,3 +94,17 @@ def update_images(self):
         files = os.listdir(directory)
         self.images = [os.path.join(directory, file) for file in files
                        if (file.endswith('.png') or file.endswith('.jpg'))]
+
+    def play(self):
+        """Start scheduling wallpapers."""
```
Comment (nkprince007) [inline uiplib/gui/mainGui.py]: Change the docstring, please.

File: uiplib/scheduler.py [hunk 2]
```diff
@@ -15,52 +15,23 @@
 except ImportError:
     # not on windows
     pass
+from threading import Thread
 
 
-class scheduler():
+class scheduler(Thread):
```
Comment (nkprince007) [inline uiplib/scheduler.py]: Inheritance. :smile_cat: awesome, bro @abhsag24

File: uiplib/gui/mainGui.py (hunk 1 above)
Comment (abhsag24) [inline uiplib/gui/mainGui.py]: any ideas what to?

**Comment by nkprince007** [comment]
Do change the shortlog of second commit, bro. `scheduling` not `scheduiling`

**Comment by nkprince007** [comment]
The shortlog of first commit looks okay, but it doesn't give away what you changed on the first look, I feel.

**Comment by abhsag24** [comment]
@nkprince007 all the changes of the first commit were asthetic. What do you want me to change it to?

**Pull request merged**